.github/ISSUE_TEMPLATE/task.yml

```diff
@@ -0,0 +1,16 @@
+name: Task
+description: A workitem that needs to be done while developing a story, feature, addressing tech debt, documentation, etc.
+labels: [task]
+
+body:
+  - type: textarea
+    attributes:
+     label: Description
+     description: A concise description of the task or suggested implementation.
```

Comment:
Nitpicking: I'm a stickler for using the word in the description/definition of something, so maybe: "A concise explanation of the task or suggested implementation."